massa-execution/src/bootstrap_state.rs

```diff
@@ -0,0 +1,46 @@
+use massa_models::{DeserializeCompact, SerializeCompact, Slot};
```

Comment:
I'm not sure about the name of this file.

Reply:
massa-execution/src/bootstrap_state.rs ? It's because it defines the BootstrapExecutionState structure.  Any better idea ?

Reply:
Maybe something like interface, or  exports? to prepare the split of all crates ?

Reply:
done